### PR TITLE
11291 add opt out modal

### PR DESCRIPTION
**Pull request by rianfowler**
[Ticket](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/11291)

screen shot 2018-07-13 at 14 17 55

download-7

### CODE REVIEW

File: content/pages/education/gi-bill.md
Comment (rianfowler) [inline content/pages/education/gi-bill.md]: these changes were pulled from [this](https://github.com/department-of-veterans-affairs/vets-website/pull/7947) content update

File: content/pages/education/index.md
Comment (rianfowler) [inline content/pages/education/index.md]: these changes were pulled from [this](https://github.com/department-of-veterans-affairs/vets-website/pull/7947) content update

File: content/pages/education/opt-out-information-sharing.md
Comment (rianfowler) [inline content/pages/education/opt-out-information-sharing.md]: most of these changes were pulled from [this](https://github.com/department-of-veterans-affairs/vets-website/pull/7947) content update

File: content/pages/education/opt-out-information-sharing.md
Comment (rianfowler) [inline content/pages/education/opt-out-information-sharing.md]: Adding a widget

File: content/pages/education/opt-out-information-sharing.md
Comment (rianfowler) [inline content/pages/education/opt-out-information-sharing.md]: Adds Opt out button at the bottom

File: src/applications/edu-benefits/components/OptOutWizard.jsx
Comment (rianfowler) [inline src/applications/edu-benefits/components/OptOutWizard.jsx]: This content is still [pending](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/11291#issuecomment-401877057).

File: src/applications/edu-benefits/components/createOptOutApplicationStatus.jsx
Comment (rianfowler) [inline src/applications/edu-benefits/components/createOptOutApplicationStatus.jsx]: I dunno if this is correct or necessary.

File: src/applications/edu-benefits/components/OptOutWizard.jsx
Comment (rianfowler) [inline src/applications/edu-benefits/components/OptOutWizard.jsx]: This url is from this [comment](https://github.com/department-of-veterans-affairs/vets-website/pull/7975#issuecomment-403982841).

File: content/pages/education/index.md
Comment (cehsu) [reply content/pages/education/index.md]: We're going to want to remove these cross link changes for now.

File: src/applications/edu-benefits/components/OptOutWizard.jsx
Comment (cehsu) [inline src/applications/edu-benefits/components/OptOutWizard.jsx]: This tel number should be formatted as an anchor according to the design, similar to as is done in the need help [component](https://github.com/department-of-veterans-affairs/vets-website/blob/43b510a621a31e73e51022661668047f84a808dd/src/applications/edu-benefits/components/GetFormHelp.jsx#L9): 
`        <a className="help-phone-number-link" href="tel:+1-888-442-4551">1-888-442-4551</a><br/>`

**Review by cehsu — APPROVED** [review]
LGTM pending a few minor changes, and @peggygannon's approval.

File: src/applications/edu-benefits/components/createOptOutApplicationStatus.jsx
Comment (cehsu) [reply src/applications/edu-benefits/components/createOptOutApplicationStatus.jsx]: Interesting question, as there are multiple types of edu forms, the edu wizard supports showing a user multiple saved forms. It doesn't seem like this form should be considered an edu form in the sense that it should be included in that list, nor that there will be multiple opt-out forms.

File: src/applications/edu-benefits/components/OptOutWizard.jsx
Comment (rianfowler) [reply src/applications/edu-benefits/components/OptOutWizard.jsx]: resolved.

File: src/applications/edu-benefits/components/createOptOutApplicationStatus.jsx
Comment (rianfowler) [reply src/applications/edu-benefits/components/createOptOutApplicationStatus.jsx]: Makes sense- removing them didn't impact any behavior or cause any errors.

File: src/applications/edu-benefits/tests/components/OptOutWizard.unit.spec.jsx
Comment (rianfowler) [inline src/applications/edu-benefits/tests/components/OptOutWizard.unit.spec.jsx]: Modal is chaining some calls here that make it tricky to mock a SUT that uses it.

**Comment by rianfowler** [comment]
@peggygannon 

- This change has the WIP branch pulled in. I'll merge this after the content changes are merged into master.
- This includes the content changes to the modal.

**Pull request merged**
